### PR TITLE
[NG][SCSS] Options item and searcher style

### PR DESCRIPTION
**Pull request by sanlucca**
Fixes style issues with option items and searcher

### CODE REVIEW

File: packages/ng/libraries/root/src/style/definitions/option/_option-searcher.scss [hunk 1]
```diff
@@ -14,8 +14,18 @@
 	.textfield-input {
 		border-radius: 0;
 		box-shadow: none;
+		background: #FFFFFF;
```
Comment (RipripKipkip) [inline packages/ng/libraries/root/src/style/definitions/option/_option-searcher.scss:17]: background or background-color (look, line 25), that is the question

Comment (BertrandPodevin) [reply packages/ng/libraries/root/src/style/definitions/option/_option-searcher.scss:17]: replace occurences of `#FFFFFF` with the recently added `_theme("colors.white");` ?

Comment (lucienbertin) [reply packages/ng/libraries/root/src/style/definitions/option/_option-searcher.scss:17]: i just checked and the colors you are refering to dont exist in the theme

https://github.com/LuccaSA/lucca-front/blob/rc/packages/scss/src/theming/_palettes.scss

is it an oversight ? should it be added to the "standard" color map ?

File: packages/scss/src/theming/_commons.scss [hunk 2]
```diff
@@ -59,7 +59,7 @@ $commons: (
 	mobile: "md",
 	loading: (
 		background: _color("primary", "lighter"),
-		frontground: _color("primary"),
+		frontground: _color("primary", "color"),
```
Comment (sanlucca) [inline packages/scss/src/theming/_commons.scss:62]: This isn't needed.

File: packages/scss/src/theming/_commons.scss [hunk 3]
```diff
@@ -59,7 +59,7 @@ $commons: (
 	mobile: "md",
 	loading: (
 		background: _color("primary", "lighter"),
```
Comment (sanlucca) [inline packages/scss/src/theming/_commons.scss:61]:
```suggestion
		background: _color("primary", "200"),
```

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1054

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1054

**Pull request merged**